data/MBS/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "MBS",
+  "symbol": "MBS",
+  "decimals": 18,
+  "description": "MonkeyLeague is the next-gen casual esports game empowering players to create, play, compete, and earn.",
+  "website": "https://monkeyleague.io",
+  "twitter": "@TheMonkeyLeague",
+  "tokens": {
+    "ethereum": {
+      "address": "0x4Bbf46E7c31eDc1F33aF4d57F0C4a0f13398427c"
+    },
+    "goerli": {
+      "address": "0x4fBb4BF42eE32A0dDDe8c531c958086A92A6980B"
+    },
```

Comment:
Now that base mainnet is launched to developers feel free to deploy there & add a "base" section to this PR while we're getting the rest of the issues figured out.

Reply:
👍 will be done so on Sunday